website/docs/docs/dbt-cloud-apis/sl-python-sdk.md

```diff
@@ -0,0 +1,150 @@
+
+If you're using async frameworks like [FastAPI](https://fastapi.tiangolo.com/) or [Strawberry](https://github.com/strawberry-graphql/strawberry), installing the sync version of the SDK will block your event loop and can significantly slow down your program. In this case, we strongly recommend using async installation.
+
+</TabItem>
+</Tabs>
+
+## Usage
+To run operations against the Semantic Layer APIs, instantiate (create an instance of) a `SemanticLayerClient` with your specific [API connection parameters](/docs/dbt-cloud-apis/sl-api-overview):
+
+```python
+from dbtsl import SemanticLayerClient
+
+client = SemanticLayerClient(
+    environment_id=123,
+    auth_token="<your-semantic-layer-api-token>",
+    host="semantic-layer.cloud.getdbt.com",
+)
+
+# query the first metric by `metric_time`
+def main():
+    with client.session():
+        metrics = client.metrics()
+        table = client.query(
+            metrics=[metrics[0].name],
+            group_by=["metric_time"],
+        )
+        print(table)
+
+main()
+```
+
+**Note**: All method calls that reach out to the APIs need to be within a `client.session()` context manager. This allows the client to establish a connection to the APIs only once and reuse the same connection between API calls. 
+
+We recommend creating an application-wide session and reuse the same session throughout the application for optimal performance. Creating a session per request is discouraged and inefficient.
+
+### asyncio usage
+If you're using asyncio, import `AsyncSemanticLayerClient` from `dbtsl.asyncio`. The APIs of `SemanticLayerClient` and `AsyncSemanticLayerClient` are identical, but the async version has async methods that you need to `await`.
+
+```python
+import asyncio
+from dbtsl.asyncio import AsyncSemanticLayerClient
+
+client = AsyncSemanticLayerClient(
+    environment_id=123,
+    auth_token="<your-semantic-layer-api-token>",
+    host="semantic-layer.cloud.getdbt.com",
+)
+
+async def main():
+    async with client.session():
+        metrics = await client.metrics()
+        table = await client.query(
+            metrics=[metrics[0].name],
+            group_by=["metric_time"],
+        )
+        print(table)
+
+asyncio.run(main())
+
+```
+
+## Integrate with dataframe libraries
+
+The SDK returns all query data as [pyarrow](https://arrow.apache.org/docs/python/index.html) tables. 
+
+The SDK library doesn't come bundled with [Polars](https://pola.rs/) or [Pandas](https://pandas.pydata.org/). If you use these libraries, add them as dependencies in your project.
```

Comment:
Maybe a little more positive language
```suggestion
If you use [Polars](https://pola.rs/) or [Pandas](https://pandas.pydata.org/) libraries, you need to add them as dependencies in your project.
```

Reply:
what about this? i prefer being explicit:

> If you use [Polars](https://pola.rs/) or [Pandas](https://pandas.pydata.org/) libraries, you need to add them as dependencies in your project.  This is because the SDK library provides you with the flexibility to choose your data processing tool and doesn't come bundled with them.

Reply:
cc @serramatutu